server/src/main/java/org/apache/iotdb/db/metadata/mnode/MeasurementMNode.java

```diff
@@ -67,12 +75,21 @@ public MeasurementMNode(
   }
 
   public MeasurementMNode(
-      MNode parent, String measurementName, IMeasurementSchema schema, String alias) {
+      IMNode parent, String measurementName, IMeasurementSchema schema, String alias) {
     super(parent, measurementName);
     this.schema = schema;
     this.alias = alias;
   }
 
+  @Override
+  public int getMeasurementMNodeCount() {
+    return 1;
```

Comment:
You can use a constant to replace it

Reply:
Maybe unnecessary. It's understandable and there may be no changes on this value.